packages/nimble-components/src/table-column/icon/tests/table-column-icon.stories.ts

```diff
@@ -76,17 +78,22 @@ export const iconColumn: StoryObj<IconColumnTableArgs> = {
             style="${isChromatic() ? '--ni-private-spinner-animation-play-state:paused' : ''}"
         >
             <${tableColumnTextTag} field-name="firstName" >
-                Name
+                First name
+            </${tableColumnTextTag}>
+            <${tableColumnTextTag} field-name="lastName" >
+                Last name
             </${tableColumnTextTag}>
             <${tableColumnIconTag} field-name="status" group-index="0">
-                Status
+                <${iconXmarkCheckTag} title="Is a Simpson"></${iconXmarkCheckTag}>
+
                 <${mappingIconTag} key="fail" icon="${iconXmarkTag}" severity="error" text="Not a Simpson"></${mappingIconTag}>
                 <${mappingIconTag} key="success" icon="${iconCheckLargeTag}" severity="success" text="Is a Simpson"></${mappingIconTag}>
                 <${mappingSpinnerTag} key="calculating" text="Calculating"></${mappingSpinnerTag}>
                 <${mappingIconTag} key="unknown" text="Unknown"></${mappingIconTag}>
             </${tableColumnIconTag}>
             <${tableColumnIconTag} field-name="isChild" key-type="boolean">
-                Is Child
+                <${iconChartDiagramChildFocusTag} title="Is child"></${iconChartDiagramChildFocusTag}>
```

Comment:
I think for maximum humor, we should group by `isChild`, properly mark Homer as "Is a Simpson", but mark Homer as "Calculating" for "Is a Child".

Reply:
I'm not against that change, but I'd rather not include that in this PR. Feel free to send one my way if you want to make those tweaks to this story.